addons/website/controllers/main.py

```diff
@@ -33,7 +33,7 @@ class Website(openerp.addons.web.controllers.main.Home):
     def index(self, **kw):
         page = 'homepage'
         try:
-            main_menu = request.registry['ir.model.data'].get_object(request.cr, request.uid, 'website', 'main_menu')
+            main_menu = request.env.ref['website.main_menu']
```

Comment:
`ref` is a method of `env`, so you better use parentheses `()` instead of brackets `[]`...